core/commands/dht.go

```diff
@@ -9,9 +9,12 @@ import (
 
 	key "github.com/ipfs/go-ipfs/blocks/key"
 	cmds "github.com/ipfs/go-ipfs/commands"
+	dag "github.com/ipfs/go-ipfs/merkledag"
 	notif "github.com/ipfs/go-ipfs/notifications"
 	path "github.com/ipfs/go-ipfs/path"
+	routing "github.com/ipfs/go-ipfs/routing"
 	ipdht "github.com/ipfs/go-ipfs/routing/dht"
+	"golang.org/x/net/context"
```

Comment:
This shouldn't be here.
Maybe we should wait to clear up Go 1.7 situation in https://github.com/ipfs/go-ipfs/pull/3110